js/addressfinder/src/main.js

```diff
@@ -0,0 +1,11 @@
+import Magento from './magento';
+
+let script = document.createElement('script');
+script.src = 'https://api.addressfinder.io/assets/v3b/widget.js';
```

Comment:
No need for the "beta" version to be used for customers. 

```suggestion
script.src = 'https://api.addressfinder.io/assets/v3/widget.js';
```